profiles/car.lua

```diff
@@ -11,6 +11,9 @@ access_tags_hierarchy = { "motorcar", "motor_vehicle", "vehicle", "access" }
 service_tag_restricted = { ["parking_aisle"] = true }
 restriction_exception_tags = { "motorcar", "motor_vehicle", "vehicle" }
 
+-- A list of suffixes to suppress in name change instructions
+suffix_list = { "N", "NE", "E", "SE", "S", "SW", "W", "NW", "road", "street" }
```

Comment:
A change from Road to Street (Avenue, Drive) is usually significant. If the problem is that a road alternates between the two every block, then that's likely a data issue.

What about prefixes? Most cities in the U.S. use prefixes such as North or South instead of suffixes to indicate the current city quadrant. There's more ambiguity with prefixes – North Bend Road could be named after North Bend – but a change from North to South in such cases would be prettg unambiguous.